src/nuget/Microsoft.DotNet.BuildTools.nuspec

```diff
@@ -1,4 +1,4 @@
-<?xml version="1.0" encoding="utf-8"?>
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think in here you changed the encoding which I suppose it's an undesired change.